cfme/tests/configure/test_register_appliance.py

```diff
@@ -175,19 +175,23 @@ def test_rh_registration(appliance, request, reg_method, reg_data, proxy_url, pr
     request.addfinalizer(appliance.unregister)
 
 
-def test_rhsm_registration_check_repo_names(temp_appliance_preconfig_funcscope, soft_assert):
+def test_rhsm_registration_check_repo_names(
+        temp_appliance_preconfig_funcscope, soft_assert, appliance):
     """ Checks default rpm repos on a fresh appliance """
     ver = temp_appliance_preconfig_funcscope.version.series()
-    # TODO We need the context manager here is that RedHatUpdates doesn't yet support Collections.
+    extras = cfme_data['redhat_updates']['repos']['pre_592'] if appliance.version < '5.9.2' else\
```

Comment:
**REQUIRED:** We don't like the use of ```\``` for line separators, please refactor the code to avoid it.